packages/core/src/enums/ImageQualityStatus.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Status of a frame as it gets loaded.  This is ordered, with lower
+ * values being more lossy, and higher values being less lossy.
+ */
+enum ImageQualityStatus {
+  /**
+   *  Replicate is a duplicated image, from some larger distance
+   */
+  FAR_REPLICATE = 1,
+
+  // Skipping a value here and after the next replicate to allow for interpolation
+  // enum values.
+
+  /**
+   * Adjacent replicate is a duplicated image of a nearby image
+   */
+  ADJACENT_REPLICATE = 3,
+
+  /**
+   * Sub resolution images are encodings of smaller than full size, and are
+   * generally lower quality than a lossy regular image.
+   */
+  SUBRESOLUTION = 6,
+  /**
+   *  Lossy images, either complete or partial
```

Comment:
Requires more documentation on how it is different from subresolution. There is no mention of "partial" anywhere here, but you are using "partial."